src/Nonlinear/operators.jl

```diff
@@ -0,0 +1,748 @@
+function _create_binary_switch(ids, exprs)
+    if length(exprs) <= 3
+        out = Expr(:if, Expr(:call, :(==), :id, ids[1]), exprs[1])
+        if length(exprs) > 1
+            push!(out.args, _create_binary_switch(ids[2:end], exprs[2:end]))
+        end
+        return out
+    else
+        mid = length(exprs) >>> 1
+        return Expr(
+            :if,
+            Expr(:call, :(<=), :id, ids[mid]),
+            _create_binary_switch(ids[1:mid], exprs[1:mid]),
+            _create_binary_switch(ids[mid+1:end], exprs[mid+1:end]),
+        )
+    end
+end
+
+let exprs = map(SYMBOLIC_UNIVARIATE_EXPRESSIONS) do arg
```

Comment:
why let blocks here and not const?

Reply:
Added a comment. We want a local variable, not something that persists in the scope of the module.